library/oneview_server_profile.py

```diff
@@ -345,7 +345,13 @@ def __present(self):
 
             self.__validations_for_os_custom_attributes(merged_data, self.current_resource.data)
 
-            if not compare(self.current_resource.data, merged_data):
+            # removed the below fields as part of idempotency checks
+            updated_data = deepcopy(merged_data)
+            updated_data.pop('initialScopeUris', None)
+            if updated_data.get('firmware'):
+                updated_data['firmware'].pop('firmwareActivationType', None)
```

Comment:
same, why removing firmwareActivationType before the comparison?

Reply:
Similarly, firmwareActivationType key is always null in every GET response and it will effect our idempotency checks. So ignored that while making idempotency checks.